src/Sarif/Sarif.csproj

```diff
@@ -39,6 +39,7 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="GlobalSuppressions.cs" />
```

Comment:
I don't know what these have to do with the PR or why GitHub is claiming the GlobalSuppressions are a binary file. Can you remove this?